### ISSUE TITLE
Tracking: open decisions and unrun experiments

### ISSUE THREAD

**Issue by ash-ding** [ash-ding/Glyph#29]
Every open decision and every unrun experiment, as of the data-layer pass on
2026-08-30. Details, measurements and options are in the linked issues; this is
the map and the order.

## Already settled (no issue)

- `decode = whiten` -- reverified at 17^3: 1163 distinct unary outputs over
  4913 inputs, top at 2.1%, inside the range whiten was validated at
- all three presets on `|V| = 17^3`, `d_digit = 16`, `d_total = 48`
- `binary_coupling = 0.25`
- generation raises with a diagnosis rather than returning a short test set
  (`5fe521e`)
- held-out pairs drawn from realizable pairs, a fixed share per shape class
  (`8253c9a`)
- the skeleton-only and table-only ceilings ship with every score, plus
  `headroom` (`8253c9a`)

## Data layer

| | decision | blocks |
|---|---|---|
| #1 | `n_structural` for pi_low -- 3, 4 or 5 | freezing the data layer |
| #2 | preset: condition or sampler | paper wording, grid organisation |
| #3 | does pi stay measured on iid only | every recorded pi value |
| #4 | ceiling as covariate, normaliser, or both | every figure |
| #5 | match comp's table demand to iid's | what `comp` measures |
| #6 | sample expression depth rather than fix it per split | the instance definition |
| #7 | `value_form` -- underscore or letter_sep | A2's economics |
| #8 | `unary_coupling` | whether pi_low is measurable at all |
| #9 | test-set size and per-arm evaluation size | every arm run |
| #10 | digit layout | couples to #7 and #8 |

## Protocol

| | decision | blocks |
|---|---|---|
| #11 | dead parameters in `synthesize_data` | what the trace means |
| #12 | `filter_data` missing -> H3 not measurable on T1 | H3 |
| #13 | `evaluate` gated behind the weights container | arm comparability |
| #14 | dev is carved from purchases and does not estimate test | what the agent optimises |
| #15 | the query cap Q | the main figure's x-axis |
| #16 | `declare_target` roles are declarative only | H3, Fig. 4 |
| #17 | implement A7 | Q2 of the research question |
| #18 | update the Phase 1 artifact's H1 wording | three documents |

## Experiments

**P1 -- decides whether the benchmark is usable**

- #19 run a trained adapter on the full test set: what is the real ceiling?
- #20 rerun A0' on a paired subset: the core evidence for H1

**P2 -- invalidated by the preset and held-pair changes**

- #21 self-check #4 (hiddenness)
- #22 self-check #5 (capacity), at the better learning rate
- #23 E0's four arms

**P3 -- not started**

- #24 unary_coupling x value_form sweep
- #25 variance decomposition
- #26 the phase diagram
- #27 prompt caching (specified in plan 6.3, never implemented, zero cache hits
  across eight runs)
- #28 T2 -- zero lines, and it carries half the paper

## Order

E0 has already been invalidated twice: once by the preset change, once by the
held-pair change. Any of #1, #5, #6, #7, #8 would make it three times.

```
1. settle #1 #6 #7 #8        CPU only, or data already in hand
2. run #19                   confirm there is a measurable band at all
3. settle #9 #4              evaluation size and how scores are reported
4. run #21 #22, then #23     one pass, not three
5. settle #11-#18            then touch the arms
```

#2, #3 and #18 are about how results are written up rather than produced. They
block nothing and can be settled last.

## Reading

`docs/progress.md` carries the measurements behind all of this, append-only.
The entry dated 2026-08-30 covers this pass, including seven corrections to
earlier conclusions in the same file.
